mlrun/projects/project.py

```diff
@@ -3816,8 +3816,10 @@ def list_artifacts(
             case-sensitive. This means that querying for ``~name`` may return artifacts named
             ``my_Name_1`` or ``surname``.
         :param tag: Return artifacts assigned this tag.
-        :param labels: Return artifacts that have these labels. Labels can either be a dictionary {"label": "value"} or
-            a list of "label=value" (match label key and value) or "label" (match just label key) strings.
+        :param labels: Return artifacts that have these labels. This can be provided as:
+            - A dictionary in the format {"label": "value"} to match specific label key-value pairs.
+            - A list of strings formatted as "label=value" to match specific label key-value pairs.
+            - A list of strings formatted as "label" to match entities with the specified label key only.
```

Comment:
```suggestion
        :param labels: Filter artifacts by labels key-value pairs or key existence. This can be provided as:
            - A dictionary in the format {"label": "value"} to match specific label key-value pairs or {"label": None} for key existence.
            - A list of strings formatted as "label=value" to match specific label key-value pairs or "label" for key existence.
            - A comma separated string formatted as "label1=value1,label2" to match entities with the specified key-value pair/key existence.
```

Reply:
We also support list of comma separated labels. Need to allow this in validation.

Reply:
> We also support list of comma separated labels. Need to allow this in validation.

It is already supported in validation because if the input is a list, we check that all elements are strings. The actual validation is performed on the server https://github.com/mlrun/mlrun/blob/61f943eeeb23ea7e3c0b4379bdf927c694359eb3/server/api/db/sqldb/db.py#L4717

Reply:
For the last bullet, it should be a list of comma-separated strings, not just a single comma-separated string.